lib/structures/CommandInteraction.js

```diff
@@ -357,10 +368,10 @@ class CommandInteraction extends Interaction {
     * @arg {Number} content.components[].type The type of component - If 1, it is a collection and a `components` array (nested) is required; if 2, it is a button; if 3, it is a select menu
     * @arg {String} [content.components[].url] The URL that the component should open for users (type 2 style 5 only)
     * @arg {String} [content.content=""] A content string
-    * @arg {Object} [content.embeds] An array of up to 10 embed objects. See [the official Discord API documentation entry](https://discord.com/developers/docs/resources/channel#embed-object) for object structure
```

Comment:
why was this removed

Reply:
Honestly not sure, its back after pulling dev tho